ctapipe/tools/tests/test_tools.py

```diff
@@ -285,6 +287,28 @@ def test_info():
     info(show_all=True)
 
 
+def test_fileinfo(tmp_path, dl1_image_file):
+    """ check we can run ctapipe-fileinfo and get results """
+    import yaml
+    from astropy.table import Table
+
+    index_file = tmp_path / "index.fits"
+    command = f"ctapipe-fileinfo {dl1_image_file} --output-table {index_file}"
+    stream = os.popen(command)
```

Comment:
Better use `ret = subprocess.run`, the os commands are low-level and discouraged for normal use.

Reply:
yes good idea. Done